knimin/lib/tests/test_data_access.py

```diff
@@ -297,6 +297,20 @@ def test_get_ag_barcode_details(self):
             del(obs[key]['registered_on'])
         self.assertEqual(obs, exp)
 
+    def test_list_ag_surveys(self):
+        self.assertItemsEqual([(-1, 'Personal Information', True),
+                               (-2, 'Pet Information', True),
+                               (-3, 'Fermented Foods', True),
+                               (-4, 'Surfers', True),
+                               (-5, 'Personal_Microbiome', True)],
+                              db.list_ag_surveys())
+        self.assertItemsEqual([(-1, 'Personal Information', False),
+                               (-2, 'Pet Information', True),
+                               (-3, 'Fermented Foods', False),
+                               (-4, 'Surfers', True),
+                               (-5, 'Personal_Microbiome', False)],
+                              db.list_ag_surveys([-2, -4]))
+
```

Comment:
I do think that this is a really critical check.  Now it looks like every time that a survey gets added, this test would need to be updated.  What do you think about this?  Do you think that it is worth while to raise an issue to make these tests - so that we don't need to update these tests every time that the surveys are updated.